packages/lightgbm/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: lightgbm
+  version: 3.3.2
+source:
+  url: https://files.pythonhosted.org/packages/8f/51/d272fa065be3be615d279be915705fa3824b86155e36c974cfb8d3ceec1e/lightgbm-3.3.2.tar.gz
+  sha256: 5d25d16e77c844c297ece2044df57651139bc3c8ad8c4108916374267ac68b64
+  patches:
+    - patches/0001-noomp-True.patch
+build:
+  script: |
+    export CMAKE_TOOLCHAIN_FILE=$PYODIDE_ROOT/packages/lightgbm/cmake/Toolchain.cmake
+  exports: "whole_archive"
```

Comment:
Does it not work with `exports: "requested"`?

Reply:
It'll fail saying "OSError: [Errno 7] Argument list too long: 'em++'"